lite/verifying/provider_test.go

```diff
@@ -250,13 +251,14 @@ func TestConcurrencyInquirerVerify(t *testing.T) {
 
 	// Initialize a Verifier with the initial state.
 	err := trust.SaveFullCommit(fcz[0])
-	require.Nil(err)
-	cert := NewDynamicVerifier(chainID, trust, source)
-	cert.SetLogger(log.TestingLogger())
+	require.NoError(err)
+	vp := NewProvider(chainID, trust, source)
+	vp.SetLogger(log.TestingLogger())
+	cp := NewConcurrentProvider(vp)
 
 	err = source.SaveFullCommit(fcz[7])
 	err = source.SaveFullCommit(fcz[8])
```

Comment:
unrelated to the changes here; but the first err should probably be checked too.